CONTRIBUTING.md

```diff
@@ -41,9 +41,9 @@ If your contribution is significantly big it is better to first check with the p
 
 This project uses [Rush](https://rushjs.io) to manage our many Azure SDK libraries within a single repository. It is highly recommended that you read the [Rush Developer Tutorials](https://rushjs.io/pages/developer/new_developer/) to familiarize yourself with the tool.
 
-While you
-can continue to contribute to the project using the standard `npm` workflow, adopting Rush will provide you many benefits:
+While you can using the standard `npm` workflow, adopting Rush will provide you many benefits:
 
+- Some of our devDependencies are not published to the public registery (.e.g. our own ESLint plugin), and Rush is configured to install them correctly.
```

Comment:
How would someone using npm and not rush handle this case?

Reply:
I do not think there is a way to handle this case with `npm` unless we publish our internal tools somewhere. We need to either maintain our own registery or publish them [privately](https://docs.npmjs.com/creating-and-publishing-private-packages).

Reply:
In that case should we avoid talking about the option of not using rush and using only npm? Or perhaps that npm is applicable to the auto generated packages and rush for the rest?

Reply:
I did an edit here: https://github.com/Azure/azure-sdk-for-js/pull/11920/commits/931130decf89f699c5aacfb1f6ff53fa40c21dde.